python_bindings/bindings.cpp

```diff
@@ -0,0 +1,1144 @@
+                         int m,
+                         double eps,
+                         double gamma,
+                         double alpha,
+                         double homophily,
+                         double reciprocity,
+                         double K,
+                         bool mean_activities,
+                         bool mean_weights,
+                         size_t n_bots,
+                         std::vector<int> bot_m,
+                         std::vector<double> bot_activity,
+                         std::vector<double> bot_opinion,
+                         std::vector<double> bot_homophily,
+                         bool use_reluctances,
+                         double reluctance_mean,
+                         double reluctance_sigma,
+                         double reluctance_eps,
+                         double covariance_factor,
+                         double friction_coefficient) {
+                 Seldon::Config::ActivityDrivenInertialSettings activity_driven_inertial_settings;
+                 if (max_iterations.has_value()) {
+                     activity_driven_inertial_settings.max_iterations = max_iterations.value();
+                 }
+                 activity_driven_inertial_settings.dt = dt;
+                 activity_driven_inertial_settings.m = m;
+                 activity_driven_inertial_settings.eps = eps;
+                 activity_driven_inertial_settings.gamma = gamma;
+                 activity_driven_inertial_settings.alpha = alpha;
+                 activity_driven_inertial_settings.homophily = homophily;
+                 activity_driven_inertial_settings.reciprocity = reciprocity;
+                 activity_driven_inertial_settings.K = K;
+                 activity_driven_inertial_settings.mean_activities = mean_activities;
+                 activity_driven_inertial_settings.mean_weights = mean_weights;
+                 activity_driven_inertial_settings.n_bots = n_bots;
+                 activity_driven_inertial_settings.bot_m = bot_m;
+                 activity_driven_inertial_settings.bot_activity = bot_activity;
+                 activity_driven_inertial_settings.bot_opinion = bot_opinion;
+                 activity_driven_inertial_settings.bot_homophily = bot_homophily;
+                 activity_driven_inertial_settings.use_reluctances = use_reluctances;
+                 activity_driven_inertial_settings.reluctance_mean = reluctance_mean;
+                 activity_driven_inertial_settings.reluctance_sigma = reluctance_sigma;
+                 activity_driven_inertial_settings.reluctance_eps = reluctance_eps;
+                 activity_driven_inertial_settings.covariance_factor = covariance_factor;
+                 activity_driven_inertial_settings.friction_coefficient = friction_coefficient;
+                 return activity_driven_inertial_settings;
+             }),
+             "max_iterations"_a = std::optional<int>{},
```

Comment:
Add a note about where the defaults are coming from so they can be updated (also for all the structures with defaults from a `config` file).